src/amo/components/AddonReviewCard/index.js

```diff
@@ -358,6 +359,11 @@ export class AddonReviewCardBase extends React.Component<InternalProps> {
     } = this.props;
 
     let byLine;
+    const addonIconURL = review ? review.reviewAddon.iconUrl : null;
+    const addonName =
+      review?.reviewAddon?.name?.['en-US'] ||
+      review?.reviewAddon?.name?.[review?.reviewAddon?.name?._default] ||
+      null;
```

Comment:
this is connected to the change to `ReviewAddonType` I assume, but why is it needed?

Reply:
This acts as a fallback when the `en-US` name is missing. We then check the language indicated by `_default` to retrieve its corresponding name. If en-US is consistently available or if add-ons never use another language though, we can skip this.

Reply:
Ah, I see what's happening now.  Okay, you're correct about that being the data structure the API returns, but we're supposed to have already simplified `name` in another part of the code - so the object from the API is already a string - and we've never noticed the bug because this issue wasn't implemented.  

In [ExternalReviewTypeBase](https://github.com/mozilla/addons-frontend/blob/master/src/amo/api/reviews.js#L12) `name: string` should be `name: LocalizedString`, and in [createInternalReview](https://github.com/mozilla/addons-frontend/blob/master/src/amo/actions/reviews.js#L78) `name: review.addon.name` should be `name: selectLocalizedContent(review.addon.name)`.  That way when it gets to this function it's already a string in the appropriate locale.